pages/index.tsx

```diff
@@ -1,23 +1,52 @@
-import HeroSection from "@/components/home/HeroSection";
-import HeadTags from "@/components/common/HeadTags";
-import Features from "@/components/home/Features";
-import Navbar from "@/components/home/Navbar";
-import Footer from "@/components/home/Footer";
+// import HeroSection from "@/components/home/HeroSection";
+// import Features from "@/components/home/Features";
+// import Navbar from "@/components/home/Navbar";
+// import Footer from "@/components/home/Footer";
 import type { NextPage } from "next";
+import TextBox from "@/components/FrontPage/textbox";
+import TopBar from "@/components/FrontPage/topbar";
+import Btn from "@/components/FrontPage/button";
+import FrontImage from "@/components/FrontPage/Combined/frontImage";
+import {
+  MainTitle,
+  CathPhrase1,
+} from "@/Components/FrontPage/Combined/textFields";
+import ShowcaseImages from "@/Components/FrontPage/Combined/showcaseImages";
+import { FeatureBoxGrid } from "@/Components/FrontPage/featureBox";
+import WriteOnceAccessAnywhere from "@/Components/FrontPage/Combined/WriteOnceAccessAnywhere";
+import UsedTechnologies from "@/Components/FrontPage/Combined/usedTechnologies";
+import FinalSection from "@/Components/FrontPage/Combined/finalSection";
+import Footer from "@/Components/FrontPage/customFooter";
+import HeadTags from "@/components/common/HeadTags";
 
 const Home: NextPage = () => {
   return (
-    <div className="flex min-h-screen flex-col overflow-y-auto bg-slate-50 text-slate-900 dark:bg-slate-900 dark:text-slate-50">
+    <div className="max-w-screen overflow-hidden bg-chalk-50 pt-32 dark:bg-midnight-300">
       <HeadTags
         title="writedown - Your public and private diary"
         description="A simple and beautiful notes app with cloud sync, markdown and offline support. Write, share, inspire."
         ogImage="https://writedown.app/og-image.png"
         ogUrl="https://writedown.app"
       />
-      <Navbar />
-      <HeroSection />
-      <Features />
-      <Footer />
+      <TopBar></TopBar>
+      <div className="mb-12 flex flex-col gap-12 text-center">
+        <MainTitle></MainTitle>
+        <Btn variant="page">Write now!</Btn>
+      </div>
+      <FrontImage></FrontImage>
```

Comment:
Empty tags must be written properly.
```suggestion
      <FrontImage />
```